build.sh

```diff
@@ -116,6 +116,8 @@ function build_systemd() {
 # TODO: Build sample config file
 mkdir -p "$DESTDIR/$sysconfdir/google-cloud-ops-agent/"
 cp "confgenerator/default-config.yaml" "$DESTDIR/$sysconfdir/google-cloud-ops-agent/config.yaml"
+mkdir -p "$DESTDIR/$sysconfdir/logrotate.d"
+cp "default-logrotate-conf" "$DESTDIR/$sysconfdir/logrotate.d/google-cloud-ops-agent"
```

Comment:
It probably doesn't make sense to pollute the top-level directory with miscellaneous configs. I would suggest creating an `etc/logrotate.d` directory at the top level, but I'll let @hsmatulisgoogle make that call…

Reply:
moved to etc/logrotate.d. Let me know if another path is preferrable.